### DISCUSSION TITLE
Fix / workaround for CH340 driver update issue

### DISCUSSION THREAD

**Discussion by arkypita** [arkypita/LaserGRBL#2443] [General]
This version include a bugfix/workaround of the [issue with CH340 USB-Serial converter driver](https://lasergrbl.com/november-2024-windows-driver-update/) simply by filtering some exception (errors) that this new driver version throw.

This should make it unnecessary to downgrade the driver.

<hr /><em>This discussion was created from the release <a href='https://github.com/arkypita/LaserGRBL/releases/tag/v7.14.0'>Fix / workaround for CH340 driver update issue</a>.</em>

**Comment by Yak503** [comment]
I know this might be off-topic, but is there a way to connect my two tree laser engraver to my iPhone via an app or other way any help would be greatly appreciated thank you

**Comment by MoneyT24** [comment]
install (4).exe isn't commonly downloaded. Make sure you trust install (4).exe before you open it.......What will I need to do to fix this problem

**Reply by AmmiHamzah** [reply]
It's just a file name, which means there are 4 files named "install.exe" in the same folder. You can just install it directly if you are sure the file comes from this github. Just execute immediately after downloading is complete, for security and certainty.

**Comment by AmmiHamzah** [comment]
> This version include a bugfix/workaround of the [issue with CH340 USB-Serial converter driver](https://lasergrbl.com/november-2024-windows-driver-update/) simply by filtering some exception (errors) that this new driver version throw.
> 
> This should make it unnecessary to downgrade the driver.
> 
> _This discussion was created from the release [Fix / workaround for CH340 driver update issue](https://github.com/arkypita/LaserGRBL/releases/tag/v7.14.0)._

I tried to engrave cardboard, but it always got cut off even though I set the power to 10% (in material database). And when I saved it in an nc file, it turned out that the laser power was 100% (s1000) when I opened the nc file. No matter how big the percentage I enter, it always changes to 100% aka s1000. How do I fix this?

**Reply by G-Arga** [reply]
Sounds like an issue with the method of code generation, though i use GCODE files not NC. You _could_ walk through the file and edit each S-value line by line...
